index.d.ts

```diff
@@ -168,13 +185,13 @@ type UncaughtExceptionFn = (err: Error) => void;
 
 type CaptureErrorCallback = (err: Error | null, id: string) => void;
 
-type FilterFn = (payload: object) => object | Falsy;
+type FilterFn = (payload: object) => object | Falsy | void;
 
 type LambdaFn =
   | ((handler: LambdaHandlerFn) => LambdaHandlerFn)
   | ((type: string, handler: LambdaHandlerFn) => LambdaHandlerFn);
-type LambdaHandlerFn = (event: object, context: object, callback: LambdaHandlerCallbackFn) => any;
-type LambdaHandlerCallbackFn = (err?: Error | null | undefined, result?: any) => void;
+type LambdaHandlerFn = (event: object, context: object, callback: LambdaHandlerCallbackFn) => any; // TODO: Is the `any` return type correct?
+type LambdaHandlerCallbackFn = (err?: Error | null | undefined, result?: any) => void; // TODO: Can `result` really be `any`?
```

Comment:
[aws-lambda](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/aws-lambda/index.d.ts#L1019-L1043) uses the `any` type, so... 🤷‍♂️

Reply:
Hmm I wonder if I somehow could depend on those types inside of this file. That way I don't have to replicate them

Reply:
Yes, but I think you might need to add aws-lambda to the dependencies, even though we don't use it for anything code-related. 🤔 

I think we can probably just change the `LambdaHandlerFn` type to return `void | Promise<any>`. Also, I _think_ the `event` needs to be `any`--it takes any arbitrary input, so it could be _anything_.

Reply:
I fixed it by importing the `aws-lambda` types directly